docs/tutorial.rst

```diff
@@ -1256,7 +1256,7 @@ as aliases, by wrapping them in ``@()``.  For example:
 
     >>> @(_banana)
     'My spoon is tooo big!'
-    >>> echo "hello" | @(lambda args, stdin=None: stdin.strip() + args[0]) world
```

Comment:
No, this wasn't always broken this probably broke in the refactor on v0.5. Thanks for catching it!